site/docs/v1/tech/messengers/index.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: doc
+title: Messengers
+description: Adding and Configuring Messengers
+---
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Messengers
+
+Messengers are used to send messages from within FusionAuth to other systems.
+
+== Set Up a Messenger
+
+In FusionAuth, you can set up three types of messengers:
+
+- link:generic-messenger[Set Up a Generic Messenger]
+- link:kafka-messenger[Set Up a Kafka Messenger]
+- link:twilio-messenger[Set Up a Twilio Messenger]
```

Comment:
Per https://github.com/FusionAuth/fusionauth-site/blob/master/DocsDevREADME.md, 
"All links should be fully-qualified and always include a slash at the end (i.e. link:/docs/v1/tech/apis/users/ not link:users)"